tests/templates/kuttl/cluster-operation/10-install-hive.yaml.j2

```diff
@@ -15,9 +15,8 @@ spec:
   clusterConfig:
     database:
       connString: jdbc:derby:;databaseName=/tmp/hive;create=true
-      user: APP
-      password: mine
       dbType: derby
+      credentialsSecret: hive-credentials
```

Comment:
Can we make this consistent (i would prefer having `credentialSecret` below `connString` as in other examples)?

Reply:
Done: 67ed2e0